src/App.tsx

```diff
@@ -194,24 +188,24 @@ const App: React.FC = () => {
 
       const numberOfItems = treeJSON[GithubData.TREE].length;
 
-      for (let index = 0; index < numberOfItems; index += 1) {
-        const item = treeJSON[GithubData.TREE][index];
-        if (item.type === IS_FILE) {
-          const SHA = item.sha;
-          const { path } = item;
-          const blobs = await fetch(
-            `${GITHUB_API_URL_PREFIX}${owner}/${repo}${GITHUB_API_BLOBS_SUFFIX}/${SHA}`
-          )
-            .then((blobs) => blobs.json())
-            .then((data) =>
-              builtInComments.push({
-                path,
-                comment: atob(data[GithubData.CONTENT]),
-              })
-            )
-            .catch((error) => alert(`Error${error}`));
-        }
-      }
+      // for (let index = 0; index < numberOfItems; index += 1) {
```

Comment:
There is way too much code inside one function btw. Every function usually has MAX 8 lines of code otherwise it's too big, I've been very lenient on this. Future reference try breaking logic down into functions so the code is more readable.